src/utils/easing.js

```diff
@@ -1,3 +1,24 @@
 export function easeOutQuadratic(t) {
   return t * (2 - t);
 }
+// Adapted from
+// https://github.com/d3/d3-ease/blob/master/src/elastic.js#L18
+const tau = 2 * Math.PI,
+  amplitude = 1,
+  period = 0.3;
+export const elasticOut = (function custom(a, p) {
+  const s = Math.asin(1 / (a = Math.max(1, a))) * (p /= tau);
+
+  function elasticOut(t) {
+    return 1 - a * Math.pow(2, -10 * (t = +t)) * Math.sin((t + s) / p);
+  }
+
+  elasticOut.amplitude = function(a) {
```

Comment:
Confused what these are for? Same goes for the self executing function, seems like you can just do:
```javascript
const tau = 2 * Math.PI,
  a = 1,
  p = 0.3,
  s = Math.asin(1 / (a = Math.max(1, a))) * (p /= tau); 
export  function elasticOut(t) {
    return 1 - a * Math.pow(2, -10 * (t = +t)) * Math.sin((t + s) / p);
}
```
(though pick better variable names or surround with {})